server/main.go

```diff
@@ -96,12 +96,35 @@ func setUpAndRunServer(portNumber int, token, ipfsGatewayUrls string) {
 	mux.Handle("/timeout", recoveryWrapper(timeoutHttpHandler(httpState)))
 
 	log.Print("Listening...")
+	serverStopCommandSetup()
 	err := http.ListenAndServe(":"+strconv.Itoa(portNumber), mux)
 	if err != nil {
 		log.Println("error listening and serving on TCP network: %w", err)
 	}
 }
 
+func serverStopCommandSetup() {
```

Comment:
Can you send the sigterm to the docker container and this be executed or do you have to exec in?

Reply:
let me check this

Reply:
nope. Docker seems to use a different mechanism to kill the ipfs-pinner process.  
Anyway, this was meant for when running natively, as the process didn't get killed with ^C.